iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/PublicNetworkAccessCheckTest.java

```diff
@@ -236,12 +236,31 @@ void shouldCheckDbForMySqlFlexibleServers() {
   }
 
   @Test
-  void shouldCheckDbForInsightsDataCollectionEndpoints() {
+  void shouldCheckInsightsDataCollectionEndpoints() {
     verify("PublicNetworkAccessCheckTest/Microsoft.Insights_dataCollectionEndpoints/test.json",
       CHECK,
       issue(11, 10, 11, 42, MESSAGE_PUBLIC_NETWORK_ACCESS));
   }
 
+  static Stream<String> shouldCheckPublicNetworkAccessSimplifiedInSiteConfig() {
+    return Stream.of(
+      "Microsoft.Web/sites",
+      "Microsoft.Web/sites/slots");
+  }
+
+  @MethodSource
+  @ParameterizedTest(name = "[${index}] should check range public IP Address for type {0}")
```

Comment:
I didn't know that you could define custom names for the tests, looks quite nice!

Reply:
It's very useful for Parameterized Test